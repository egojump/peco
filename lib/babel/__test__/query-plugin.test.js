const babel = require('@babel/core')

test('queryPageByPath', () => {
  const { code } = babel.transform(
    `
  import { queryPageByPath } from 'peco'

  const page = queryPageByPath('/foo')
  `,
    {
      plugins: [require.resolve('../query-plugin')],
      babelrc: false
    }
  )

  expect(code).toMatchSnapshot()
})
