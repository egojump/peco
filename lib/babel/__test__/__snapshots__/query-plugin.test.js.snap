// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`queryPageByPath 1`] = `"const page = require(\\"@noop-query?type=queryPageByPath&value=/foo\\").default;"`;
